samtranslator/utils/utils.py

```diff
@@ -31,6 +31,13 @@ def __init__(self, relative_path: str) -> None:
             super().__init__("It should be a map")
 
 
+def truncate_with_suffix(s: str, length: int, suffix: str) -> str:
```

Comment:
If this needs to be a generic util, can we move to `internal` instead?

Reply:
Do you mean the whole utils file?

Reply:
Just this function (FWIW originally I just meant to add a private function in `events.py`, but generic function works as well).

Reply:
Moving it back to `events.py` as a private function.